src/runtime/analytics-service.js

```diff
@@ -315,7 +331,7 @@ export class AnalyticsService {
     const meta = new AnalyticsEventMeta();
     meta.setEventOriginator(event.eventOriginator);
     meta.setIsFromUserAction(!!event.isFromUserAction);
-
+    this.updateContextTimestamp_();
```

Comment:
Since it's called just once, and it's a one liner, could you inline it for simplicity ?

Reply:
Done.